gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfiguration.java

```diff
@@ -533,4 +548,24 @@ private static CooperativeLockingOptions getCooperativeLockingOptions(Configurat
             GCS_COOPERATIVE_LOCKING_MAX_CONCURRENT_OPERATIONS.get(config, config::getInt))
         .build();
   }
+
+  @VisibleForTesting
+  static Class<? extends AuthorizationHandler> getAuthorizationHandlerClass(Configuration config) {
+    Class<? extends AuthorizationHandler> handlerClass;
+    try {
+      handlerClass =
+          GCS_AUTHORIZATION_HANDLER_IMPL.get(
+              config,
+              (lookupKey, defaultValue) ->
+                  config.getClass(lookupKey, defaultValue, AuthorizationHandler.class));
+    } catch (RuntimeException e) {
+      if (Throwables.getRootCause(e) instanceof ClassNotFoundException) {
+        throw new RuntimeException(
+            "Can not find user specified authorization handler class.", e);
+      }
+      throw e;
+    }
```

Comment:
There no need to catch runtime exception - `ClassNotFoundException` with logged stack-trace should be sufficient for troubleshooting. Without this catch whole method could be inlined.